src/test/kotlin/minesweeper/domain/BoomPositionMakerTest.kt

```diff
@@ -0,0 +1,51 @@
+package minesweeper.domain
+
+import minesweeper.inputdata.MineGameConfig
+import minesweeper.inputdata.PositiveNumber
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+
+internal class BoomPositionMakerTest {
+
+    @Test
+    @DisplayName("입력한 폭탄 개수만큼 지뢰 위치를 생성한다")
+    fun makePosition() {
+        val mineGameCondition = MineGameConfig(
+            width = PositiveNumber(3),
+            height = PositiveNumber(3),
+            mineCount = PositiveNumber(6)
+        )
```

Comment:
생성자의 수가 늘어날수록 클래스를 사용하기 위한 유연성은 높아집니다.

```suggestion
        val mineGameCondition = MineGameConfig(
            width = 3,
            height = 3,
            mineCount = 6
        )
```